### DISCUSSION TITLE
AI

### DISCUSSION THREAD

**Discussion by yng3** [community/community#45656] [Other Feature Feedback, Questions, & Ideas] [answered]
### Artificial Intelligence

With AI technology on the rise, what are some changes that could happen to coding? AI has the ability to code almost instantly, but with many bugs. Are you concerned or are you excited?

AI is pretty much inevitable, as corny as it sounds. How can developers stay on top of their game?

**Comment by jge162** [comment, marked as answer]
AI can never replace a programmer. It is not capable of doing complex coding that requires a good programmer. Meaning it cant handle multiple files, see real time how the code is running on a device and/or debug the code. Thats just how I see it. Boots on the ground per se.

**Reply by yng3** [reply]
It seems AI could bring power to the programmers.. and big tech could fall.

**Comment by ahmadali-jamali** [comment]
I think, at first, we should determine the border between complex and not complex code. But, I believe that with genetic programming and Random processes, AI could be replaced a basic programmer in the future.

**Reply by yng3** [reply]
interesting.. complex and simple code may become obsolete considering ai and read and right code based off human language

**Reply by ahmadali-jamali** [reply]
Exactly

**Reply by ahmadali-jamali** [reply]
Exactly

**Reply by ahmadali-jamali** [reply]
Exactly
